### ISSUE TITLE
Platypus: missing `lang` attribute

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/platypus#476]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the `<html>` element is missing a `lang` attribute.

---

https://dequeuniversity.com/rules/axe/3.5/html-has-lang

---

**User impact:** Serious
**Disabilities affected:** Blind, Deafblind, Cognitive
**Requirement(s):** WCAG 2.0 (A): MUST
**WCAG Success Criteria:** 3.1.1 Language of Page

**Comment by micpap25** [comment]
Hi, this can be closed as the `html` tag does have the `lang` attribute now. 
<img width="226" alt="image" src="https://user-images.githubusercontent.com/34077605/212335830-a9669406-56ef-42da-9933-a794cc6a1229.png">

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com).
